src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_dropdown.scss

```diff
@@ -73,21 +73,21 @@ A basic dropdown that is currently used for resource actions.
 
 // The dropdown menu (ul)
 .dropdown-menu {
-    border-radius: 3px;
+    border-radius: 2px;
     float: left;
     min-width: 120px;
     padding: 0;
     margin: 5px 0 0;
     background-color: $color-text-inverted;
-    border: 1px solid $color-text-introvert;
+    border: 0.5px solid $color-structure-normal;
```

Comment:
`0.5px`?? How is that supposed to work?

https://stackoverflow.com/questions/20116132/half-pixel-in-border-width-size-it-is-not-showing